gui/src/app/state/spend/step.rs

```diff
@@ -399,6 +409,17 @@ impl Step for DefineSpend {
                         self.batch_label.valid = label.len() <= 100;
                         self.batch_label.value = label;
                     }
+                    view::CreateSpendMessage::Clear => {
```

Comment:
I think we also need to clear the MAX checkbox and unset the amount left to select:

![image](https://github.com/wizardsardine/liana/assets/121959000/bec6000a-6bc2-4a35-b06f-5c94e26fe379)

Note here the "Next" button is still clickable and will cause a panic. So I guess `is_valid` also needs to be set to `false`.

Similarly for `is_duplicate`:

![image](https://github.com/wizardsardine/liana/assets/121959000/6ed4cbde-a6cf-4a26-8cfa-2241845118f4)

Perhaps for these latter values, we could let the `redraft` and `check_valid` functions update them at the end instead of using `return Command::none()`, so only the MAX checkbox would need be cleared here.